src/Command/Docker/InitializeIbexa.php

```diff
@@ -0,0 +1,173 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license   For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace eZ\Launchpad\Command\Docker;
+
+use eZ\Launchpad\Console\Application;
+use eZ\Launchpad\Core\Client\Docker;
+use eZ\Launchpad\Core\Command;
+use eZ\Launchpad\Core\DockerCompose;
+use eZ\Launchpad\Core\ProcessRunner;
+use eZ\Launchpad\Core\ProjectStatusDumper;
+use eZ\Launchpad\Core\ProjectWizardIbexa;
+use eZ\Launchpad\Core\TaskExecutor;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Filesystem\Filesystem;
+
+class InitializeIbexa extends Command
```

Comment:
that's too bad we cannot mutualize a bit more that command with the eZ one.

Reply:
@Plopix refacotring done.